README.md

```diff
@@ -18,27 +18,27 @@ This page provides instructions for using the Analytics Toolkit space shuttle de
 * The space-shuttle-demo application fetches anomalies (classes different than 1) several times per minute from InfluxDB and then displays them.
 
 ## Deploying application to TAP
-   
+
 ### Manual deployment
-1. Upload the model to HDFS: 
-   * Download the pre-packaged model from: [https://s3.amazonaws.com/trustedanalytics/v0.7.1/models/space-shuttle-model.tar](https://s3.amazonaws.com/trustedanalytics/v0.7.1/models/space-shuttle-model.tar)
+1. Upload the model to HDFS:
+   * Download the pre-packaged model from: [https://s3.amazonaws.com/trustedanalytics/v0.7.4/models/spaceshuttleSVMmodel.zip](https://s3.amazonaws.com/trustedanalytics/v0.7.4/models/spaceshuttleSVMmodel.zip)
    * Login to the TAP console and navigate to  **Data catalog > Submit Transfer**.
    * Select input type: **Local path** and choose the previously downloaded model.
    * Enter a title in the **Title** field.
-   * Click the **Upload** button. 
-   
+   * Click the **Upload** button.
+
    Alternatively, you can create the TAP Analytics Toolkit model yourself. Refer to the [instructions](#creating-tap-analytics-toolkit-model) later on this page.
```

Comment:
I see the text shifting between ATK and Spark-tk in different places. I suspect you want to change references to ATK to Spark-tk throughout. I will flag the places I see. This sentence refers to ATK. You should probably walk through the text yourself to catch any references I might have missed.